lib/stripe_lib/models.py

```diff
@@ -358,9 +358,11 @@ def change_subscription_plan(self, subscription_id, new_price_or_plan, prorate=T
             item = {
                 'id': stripe_subscription['items']['data'][0]['id'],
             }
-            if type(new_price_or_plan) == str:
+            if isinstance(new_price_or_plan, str):
                 item['price'] = new_price_or_plan
-            elif type(new_price_or_plan) == dict:
+            elif isinstance(new_price_or_plan, type(u'')):
```

Comment:
## Reviewer Note:
Supports `unicode` type as compatible for python 2 and python 3

Used `isinstance` instead of comparing `type` on both RHS and LHS to avoid lint issue